ext/strings_test.go

```diff
@@ -105,6 +106,27 @@ var stringTests = []struct {
 	{expr: `['x', 'y'].join('-') == 'x-y'`},
 	{expr: `[].join() == ''`},
 	{expr: `[].join('-') == ''`},
+	// Escaping tests.
+	{expr: `strings.quote("first\nsecond") == "\"first\\nsecond\""`},
```

Comment:
Please add an empty string test case to ensure this function is never broken for that edge case.

Reply:
Gotcha, fixed